src/main/java/org/eclipse/xpanse/tofu/maker/api/controllers/OpenTofuMakerTaskResultApi.java

```diff
@@ -41,7 +55,52 @@ public class OpenTofuMakerTaskResultApi {
     public ResponseEntity<OpenTofuResult> getStoredTaskResultByRequestId(
             @Parameter(name = "requestId", description = "id of the request")
                     @PathVariable("requestId")
-                    String requestId) {
+                    UUID requestId) {
         return openTofuResultPersistenceManage.retrieveOpenTofuResultByRequestId(requestId);
     }
+
+    /**
+     * Batch retrieve stored openTofu results.
+     *
+     * @param requestIds list of request ids
+     * @return list of re-fetch results
+     */
+    @Tag(name = "RetrieveOpenTofuResult", description = "APIs for manage the task form tofu-maker.")
```

Comment:
i just realized that this description is also not so good. Kindly change it for all methods

"APIs to manually fetching task results from tofu-maker .